EventsExpress/Controllers/EventController.cs

```diff
@@ -27,13 +29,15 @@ public class EventController : ControllerBase
         private readonly IEventService _eventService;
         private readonly IMapper _mapper;
         private readonly ISecurityContext _securityContextService;
+        private readonly IValidator<EventViewModel> _validator;
 
-        public EventController(IEventService eventService, IMapper mapper, ISecurityContext securityContextService, IPhotoService photoService)
+        public EventController(IValidator<EventViewModel> validator, IEventService eventService, IMapper mapper, ISecurityContext securityContextService, IPhotoService photoService)
         {
             _photoService = photoService;
             _eventService = eventService;
             _mapper = mapper;
             _securityContextService = securityContextService;
+            _validator = validator;
```

Comment:
Probably it would be better to use method-located DI since this is used in a single place... @iuriikhrystiuk, thoughts?

`public async Task<IActionResult> Publish(Guid eventId, [FromServices]IValidator<EventViewModel> validator)`

Reply:
done